assets/app/view/game/sell_shares.rb

```diff
@@ -12,6 +12,9 @@ class SellShares < Snabberb::Component
       needs :action, default: Engine::Action::SellShares
 
       def render
+        step = @game.round.active_step
+        return unless step.current_actions.include?('sell_shares')
```

Comment:
Don't render unless the action is included. I'm not sure why this guard didn't already exist. Other stock round action rendering logic has it.